src/Plugin/migrate/process/SmartDateAllDayAdjust.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Drupal\stanford_migrate\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Adjust an ending timestamp to work with the "All Day" for smart date module.
+ *
+ * @code
+ * process:
+ *   field_date/end_value:
+ *     plugin: datetime_adjust
+ *     source: end_value
+ *     start_time: start_value
+ * @endcode
+ *
+ * @MigrateProcessPlugin(
+ *   id = "smartdate_adjust"
+ * )
+ */
+class SmartDateAllDayAdjust extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    if (empty($this->configuration['start_time'])) {
```

Comment:
This check is redundant since we're checking the exact same thing just below.

Reply:
This checks if if `start_time` is even populated with a value. this prevents a php warning like `Undefined index` in the following lines, so though it appears redundant, it reduces warning messages.